src/Node.hpp

```diff
@@ -83,24 +83,24 @@ class Node
 
 
   template <typename T>
-  Publisher<T> create_publisher(CanardMicrosecond const tx_timeout_usec);
+  Publisher<T> create_publisher(CanardMicrosecond const tx_timeout_usec, CanardPriority const tx_priority = CanardPriorityNominal);
```

Comment:
Hi @bakul14 :coffee: :wave: thank you very much for your contribution :rocket: . I do not prefer using default argument values, I'd rather do function overloading with another function exposing an additional parameter `tx_priority`. Can you please amend your PR accordingly?

Reply:
Ping @bakul14 :coffee: :wave: - just checking in if you are still around - if not I'll have to close this PR unmerged.

Reply:
Yes, I remember about this PR. I think I'll do it next week.